src/main/resources/org/jenkinsci/plugins/scriptler/ScriptlerManagement/sidepanel.jelly

```diff
@@ -31,13 +31,17 @@ THE SOFTWARE.
   <l:side-panel>
     <l:tasks>
       <l:task icon="images/24x24/setting.gif" href="${rootURL}/manage" title="${%Manage Jenkins}" permission="${app.ADMINISTER}" />
-      <l:task icon="images/24x24/next.gif" href="${rootURL}/scriptler" title="${%Run/Edit Scripts}" />
-      <l:task icon="images/24x24/new-package.gif" href="scriptsettings" title="${%Add a new Script}" permission="${app.ADMINISTER}" />
-      <j:if test="${!it.disableRemoteCatalog()}">
-      		<l:task icon="images/24x24/folder.gif" href="catalog" title="${%Remote Script catalogs}" permission="${app.ADMINISTER}" />
+      <j:if test="${it.hasAtLeastOneScriptlerPermission()}">
+        <l:task icon="images/24x24/next.gif" href="${rootURL}/scriptler" title="${%Run/Edit Scripts}" />
       </j:if>
-      <l:task icon="images/24x24/setting.gif" href="settings" title="${%Scriptler settings}" permission="${app.ADMINISTER}" />
-      <l:task icon="plugin/scriptler/images/Git-Icon-1788C.png" href="${rootURL}/scriptler.git" title="${%Git repo}" permission="${app.ADMINISTER}" />
+      <l:hasPermission permission="${it.scriptlerConfigure}">
+        <l:task icon="images/24x24/new-package.gif" href="scriptSettings" title="${%Add a new Script}" />
```

Comment:
The view urls were confusing for the application (two were highlighted), so I use camelCase to avoid this "issue".
![screenshot_2018-01-11_224047_001](https://user-images.githubusercontent.com/2662497/34849017-3b1e705c-f721-11e7-8898-1318d72e8dd9.png)